packages/app-store/stripepayment/api/paymentCallback.ts

```diff
@@ -61,16 +64,15 @@ async function getHandler(req: NextApiRequest, res: NextApiResponse) {
       });
     } catch (error) {
       console.error(error);
-      return {
+      throw new HttpError({
+        statusCode: 400,
         message:
           "We have received your payment. Your premium username could still not be reserved. Please contact support@cal.com and mention your premium username",
-      };
+      });
     }
   }
   callbackUrl.searchParams.set("paymentStatus", checkoutSession.payment_status);
   return res.redirect(callbackUrl.toString()).end();
 }
 
-export default defaultHandler({
-  GET: Promise.resolve({ default: defaultResponder(getHandler) }),
-});
+export default handler;
```

Comment:
removed defaultHandler and defaultResponder, everything is already handled in `apps/web/pages/api/integrations/[...args].ts`

Reply:
I think we shouldn't remove it. It takes care of some other things that has to be manually handled now and could potentially handle more things.
e.g. it was automatically erroring out when some other method than GET was used.

Reply:
I'll run some test trying to keep defaultHandler, but right now we are handling the response in too many places so probably don't need default responder